Formula/adr-tools.rb

```diff
@@ -0,0 +1,21 @@
+class AdrTools < Formula
+  desc "CLI tool for working with Architecture Decision Records."
+  homepage "https://github.com/npryce/adr-tools"
+  url "https://github.com/npryce/adr-tools/archive/2.1.0.tar.gz"
+  sha256 "1ef028cfeaa1b262a5c62845aa8965be169705370983f9ff73b17ec77bf75f70"
+
+  def install
+    # Override adr-config with custom config to point to homebrew dirs
+    system "echo '#!/bin/bash' > src/adr-config"
+    system "echo 'echo adr_bin_dir=\"#{bin}\"' >> src/adr-config"
+    system "echo 'echo adr_template_dir=\"#{prefix}\"' >> src/adr-config"
```

Comment:
As I'm trying to overwrite the `src/adr-config` file, using `Pathname#write` I end up with an error:

```
Error: Will not overwrite /private/tmp/adr-tools-20170501-58028-1kqrmh5/adr-tools-2.1.0/src/adr-config
```

Is there a suggested way to override an existing file? Or at least end up with different content for an existing file? Looking through the [Formula-Cookbook](https://github.com/Homebrew/brew/blob/master/docs/Formula-Cookbook.md) I see [inreplace](https://github.com/Homebrew/brew/blob/master/docs/Formula-Cookbook.md#inreplace). I think that might be a better solution here?

-----

For:
> Please ask upstream how we could specify this config without overriding their file.

There was [an attempt](https://github.com/Homebrew/homebrew-core/pull/3947) to put a previous version of this tool in homebrew in which a similar discussion took place. [Upstream work was done resulting in](https://github.com/npryce/adr-tools/blob/master/doc/adr/0007-invoke-adr-config-executable-to-get-configuration.md#consequences):

> Packagers can write their own implementation of adr-config that outputs configuration that matches the platform's installation conventions, and deploy it next to the adr script.